contrib/verify-commits/verify-commits.py

```diff
@@ -150,6 +155,7 @@ def main():
 
         prev_commit = current_commit
         current_commit = parents[0]
+        commit_number += 0
```

Comment:
This will never log since it doesn't increment

Reply:
Wow! Stupid mistake, haha. Thanks for pointing that out.